service/routes.py

```diff
@@ -4,8 +4,8 @@
 Service is used to manage products in the inventory.
 """
 
-# pylint: disable=cyclic-import, import-error
-from flask import jsonify, request, url_for, make_response, abort
+# pylint: disable=cyclic-import, import-error, line-too-long
+from flask import request, abort
 # pylint: disable=unused-import
 from flask_restx import Api, Resource, fields, reqparse, inputs  # noqa: F401
```

Comment:
I added `# pylint: disable=unused-import` and `# noqa: F401` because I included packages that I know we will use for refactoring (but I am not taking the refactoring issues). You can delete these once you use them.